### ISSUE TITLE
Use repeat_group for better label placement experience

### ISSUE THREAD

**Issue by tallytalwar** [tangrams/eraser-map#141]
Now that https://github.com/tangrams/tangram-es/pull/465 got merged in ES and equivalent was available on JS for some time now, we should have the eraser-map style sheet use this feature to get better placed labels.

cc. @nvkelso, @sensescape

**Comment by nvkelso** [comment]
Agree we should update style for `repeat_group`, exciting!

(The default `repeat_distance` on Tangram JS was sufficient for general purpose, assume the same will be true in Tangram ES.)

**Comment by tallytalwar** [comment]
Yup... we use the same `256px` default. Results can be seen in the latest build of eraser-map :D.

**Comment by nvkelso** [comment]
Related: https://github.com/tangrams/eraser-map/issues/57

**Comment by nvkelso** [comment]
Wow, huge improvement! Looks like we will want to optimize for less repeat, need to balance how it looks on ES with the different text engine from JS. That work should be focused in #57, this issue on `repeat_group`.
